wallet/pskt/src/bundle.rs

```diff
@@ -146,13 +146,15 @@ pub fn script_addr(script_sig: &[u8], prefix: kaspa_addresses::Prefix) -> Result
     extract_script_pub_key_address(&pay_to_script_hash_script(script_sig), prefix).map_err(Error::P2SHExtractError)
 }
 
-pub fn unlock_utxos(
+pub fn unlock_utxos_fee_per_transaction(
```

Comment:
I think it would be enough to rename the parameter instead of the whole function. But up to you

Reply:
Done and function renaming was reverted.